tests/e2e/common.go

```diff
@@ -351,24 +338,70 @@ func (suite *CCVTestSuite) commitSlashPacket(ctx sdk.Context, packetData ccv.Sla
 }
 
 // incrementTimeBy increments the overall time by jumpPeriod
+// while updating to not expire the clients
 func incrementTimeBy(s *CCVTestSuite, jumpPeriod time.Duration) {
```

Comment:
Can this be just `incrementTime` so the other function can be `incrementTimeWithoutClientUpdate`?